app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchGreetings } from '../Store/greetingsSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const { greetings } = useSelector((state) => state.greetings);
+
+  const randomGreetings = () => {
+    dispatch(fetchGreetings());
+  };
+
+  useEffect(() => {
+    randomGreetings();
```

Comment:
- [ ][OPTIONAL] You call the `randomGreetings()` every time the component is loaded, this makes the app display the greeting message too fast and consumes a lot of resources